### PR TITLE
Update import subscribers test [MAILPOET-2966]

### PR DESCRIPTION
**Pull request by veljkho**
I'd like to commit an update of the existing test case SubscriberManageImportExportCest.php to include scenario of creating a new subscribers list plus searching and selecting while importing subscribers.

Jira -> [MAILPOET-2966](https://mailpoet.atlassian.net/browse/MAILPOET-2966)

### CODE REVIEW

File: tests/acceptance/SubscriberManageImportExportCest.php [hunk 1]
```diff
@@ -101,6 +101,21 @@ private function chooseListAndConfirm(\AcceptanceTester $i) {
...
 
+  private function createNewListAndConfirm(\AcceptanceTester $i) {
+    $newListName = 'Simple List';
+    // first create a new list
+    $i->click('a.mailpoet_create_segment');
+    $i->fillField('input#new_segment_name', $newListName);
+    $i->fillField('textarea#new_segment_description', 'This is just a simple list.');
+    $i->click('input#new_segment_process');
+    // trigger dropdown to display selections and search for recently created list
+    $i->waitForElementVisible('input.select2-search__field');
```
Comment (costasovo) [inline tests/acceptance/SubscriberManageImportExportCest.php:112]: We have a helper function for selecting in select2. I think you can use `$i->selectOptionInSelect2($newListName);`
See https://github.com/mailpoet/mailpoet/blob/master/tests/_support/AcceptanceTester.php#L122

File: tests/acceptance/SubscriberManageImportExportCest.php [hunk 2]
```diff
@@ -101,6 +101,21 @@ private function chooseListAndConfirm(\AcceptanceTester $i) {
...
+    $i->fillField('input.select2-search__field', $newListName);
+    $i->click(['xpath' => '//*[@id="select2-mailpoet_segments_select-results"]/li[1]']);
+    $i->click('.mailpoet_data_manipulation_step [data-automation-id="import-next-step"]');
```
Comment (costasovo) [inline tests/acceptance/SubscriberManageImportExportCest.php:115]: You don't need to combine those `[data-automation-id="import-next-step"]` should be enough. We add those data-automation-id attributes to have stable selectors for acceptance tests.

File: tests/acceptance/SubscriberManageImportExportCest.php [hunk 3]
```diff
@@ -101,6 +101,21 @@ private function chooseListAndConfirm(\AcceptanceTester $i) {
     $i->waitForText('Import again');
   }
 
+  private function createNewListAndConfirm(\AcceptanceTester $i) {
+    $newListName = 'Simple List';
+    // first create a new list
+    $i->click('a.mailpoet_create_segment');
```
Comment (costasovo) [inline tests/acceptance/SubscriberManageImportExportCest.php:107]: Can you use $i->click(`Create a new list`) here? It is closer to user's context.

Comment (veljkho) [reply tests/acceptance/SubscriberManageImportExportCest.php:107]: Sure, I just changed to your solution. Thanks.

File: tests/acceptance/SubscriberManageImportExportCest.php (hunk 2 above)
Comment (veljkho) [reply tests/acceptance/SubscriberManageImportExportCest.php:115]: I agree, I just copy/paste that code from existing, so I now changed both mine and another inside. Thanks.

**Pull request merged**